src/db.py

```diff
@@ -62,28 +62,23 @@ def get_courses_for_student(netid):
 	courses = mongo.db.courses.find({"student_ids": netid})
 	return list(courses)
 
-
+# change this func
 def get_courses_for_staff(netid):
-	courses = mongo.db.courses.find({"staff_ids": netid})
+	courses = mongo.db.courses.find( {f'staff.{netid}': { '$exists': True } } )
```

Comment:
nit: use all double quote or single quote. Since our codebase currently use double quote, let's also change these to double quotes. A few more similar ones below.